dataverse/orgsvc/C#-NETCore/xMultipleSamples/appsettings.json

```diff
@@ -2,9 +2,10 @@
   "ConnectionStrings": {
     "default": "AuthType=OAuth;Url=https://myorg.crm.dynamics.com;Username=someone@myorg.onmicrosoft.com;RedirectUri=http://localhost;AppId=51f81489-12ee-4a9e-aaae-a2591f45987d;LoginPrompt=Auto"
   },
```

Comment:
I understand that it probably makes sense to have settings in appssettings.json, but all the other settings to control the behavior of the sample are in the Settings.cs class.

Can you move this setting there?

Also, I want the default value to be false. 
If someone wants to test elastic tables, they can set it to true.